backup-pruner.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: create dirs
+  file:
+    name: "{{ item }}"
+    owner: "root"
+    group: "root"
+    mode: "700"
+    state: "directory"
+  with_items:
+    - "/etc/tarsnap"
+    - "/var/cache"
+
+# Note: tarsnapper requires keys to end with a newline
+# which are stripped from ansible variables.
+# tarsnap.key template exists only to add that newline.
+- name: upload keys
+  template:
+    src: tarsnap.key
+    dest: "{{ item.keypath }}"
+    owner: "root"
+    group: "root"
+    mode: "400"
+  no_log: true
+  with_items: "{{ keys_to_upload }}"
+
+- name: create cache dirs
+  file:
+    name: "{{ item.cache }}"
+    owner: "root"
+    group: "root"
+    mode: "700"
+    state: directory
+  no_log: true
+  with_items: "{{ keys_to_upload }}"
+
+# NOTE: Pip doesn't work with tarsnapper
+- name: install tarsnapper
+  command: easy_install "{{ TARSNAPPER_VERSION }}"
```

Comment:
In `pmbauer.tarsnap` tarsnap version is not configurable, plus this role does bunch of things we don't really need --- like running backups in cron. It's more robust to just install tarsnapper here manually that adding bunch of swiches to `pmbauer.tarsnap` IMO.